cli/commands/hclfmt/action_test.go

```diff
@@ -258,6 +258,38 @@ func TestHCLFmtFile(t *testing.T) {
 	}
 }
 
+func TestHCLFmtStdin(t *testing.T) {
+
+	realStdin := os.Stdin
+	realStdout := os.Stdout
+
+	tempStdoutFile, err := os.CreateTemp(os.TempDir(), "stdout-*.hcl")
```

Comment:
Thank you for feedback!

Changed to `t.TempDir()`

However needed to add `tempStdoutFile.Close()` since without it test failed with ` FAIL: TestHCLFmtStdin (1.67s)
    c:\Users\username\projects\opensource\terragrunt\cli\commands\hclfmt\testing.go:1231: TempDir RemoveAll cleanup: remove C:\Users\username\AppData\Local\Temp\TestHCLFmtStdin3097231684\001\stdout.hcl4059971684: The process cannot access the file because it is being used by another process.`